src/js/containers/Login.jsx

```diff
@@ -0,0 +1,52 @@
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router'
+import { UserManager, Log, WebStorageStateStore } from 'oidc-client'
+//import { redirectAuth, processAuth } from '../actions'
+import Login from '../components/Login.jsx'
+import userManager from '../UserManager.js'
+
+
+class LoginContainer extends Component {
+  constructor(props) {
+      super(props)
+  }
+
+  componentDidMount() {
+    const manager = new UserManager({
+      authority: 'https://192.168.99.100:8443/auth/realms/hmda',
```

Comment:
Yeah this was actually a holdover from a different way of modeling the state that I overlooked. Removed in subsequent commit.